src/index.ts

```diff
@@ -125,17 +135,9 @@ export class CapBase<T> {
       });
     }
 
-    const router = "router" in args ? args.router : await CapRouter.init({ host: args.routerHost, canisterId: args.routerCanisterId });
```

Comment:
@aleacevedo check this out, I removed this piece. Create actor should not be async

Reply:
Why create actor should not be async?

Reply:
```
    static createActor<T = Record<string, ActorMethod>>(interfaceFactory: IDL.InterfaceFactory, configuration: ActorConfig): ActorSubclass<T>;

```

This is the definition of createActor in agent-js. We are promisifying something that doesn't need to be. Also with making it async we sacrifice having to createActors in class constructors (can't be async)